mindsdb_native/libs/backends/lightwood.py

```diff
@@ -87,10 +87,7 @@ def _create_lightwood_config(self):
         config['input_features'] = []
         config['output_features'] = []
 
-        for col_name in self.transaction.input_data.columns or col_name not in self.transaction.lmd['stats_v2']:
-            if col_name in self.transaction.lmd['columns_to_ignore']:
-                continue
-
+        for col_name in self.transaction.input_data.columns:
```

Comment:
shouldn't `columns_to_ignore` be skipped?

Reply:
They are dropped from input data before that